pydantic/types.py

```diff
@@ -606,3 +607,73 @@ def _get_brand(card_number: str) -> PaymentCardBrand:
         else:
             brand = PaymentCardBrand.other
         return brand
+
+
+BYTE_SIZES = {
+    'B': 1,
+    'kB': 10 ** 3,
+    'MB': 10 ** 6,
+    'GB': 10 ** 9,
+    'TB': 10 ** 12,
+    'PB': 10 ** 15,
+    'EB': 10 ** 18,
+    'KiB': 2 ** 10,
+    'MiB': 2 ** 20,
+    'GiB': 2 ** 30,
+    'TiB': 2 ** 40,
+    'PiB': 2 ** 50,
+    'EiB': 2 ** 60,
+}
+BYTE_SIZES.update({k.lower(): v for k, v in BYTE_SIZES.items()})
```

Comment:
surely we don't need this, just define them as lower case in the main dictionary?